### PR TITLE
Bugfix 6192/VerseCheck will show error message if original language quote cannot be matched

### PR DESCRIPTION
**Pull request by PhotoNomad0**
- VerseCheck will show error message if original language quote cannot be matched.

### CODE REVIEW

File: src/helpers/checkAreaHelpers.js [hunk 1]
```diff
@@ -15,7 +15,18 @@ export function getAlignedGLText(currentProjectToolsSelectedGL, contextId, bible
...
   }
-  return contextId.quote;
+  const origLangQuote = getQuoteAsString(contextId.quote);
+  const message = translate("quote_invalid", {quote: origLangQuote});
+  return message;
+}
+
+export function getQuoteAsString(quote) {
+  if (Array.isArray(quote)) {
+    return quote.map(({word}) => word).join(' ');
```
Comment (mannycolon) [inline src/helpers/checkAreaHelpers.js]: I don't think you added support for punctuation

Comment (PhotoNomad0) [reply src/helpers/checkAreaHelpers.js]: Hmm...I copied this from tC - thought this was what you and Rich had agreed on.  OK found the method flattenQuote().

Comment (PhotoNomad0) [reply src/helpers/checkAreaHelpers.js]: OK, moved method from tCore to this module for sharing.

Comment (richmahn) [reply src/helpers/checkAreaHelpers.js]: Does this make it more standard function? Cool.

**Pull request merged**